src/nyc_trees/apps/event/templates/event/partials/event_section.html

```diff
@@ -1,4 +1,6 @@
 {% with event=info.event %}
+{% block extra_markup %}
+{% endblock extra_markup %}
```

Comment:
Are you adding this because you need null out what's inherited from the parent? If so, a template comment would go a long way.